src/build/arg/mod.rs

```diff
@@ -310,8 +311,16 @@ impl<'help> Arg<'help> {
     ///
     /// assert!(m.is_present("cfg"));
     /// ```
-    pub fn long(mut self, l: &'help str) -> Self {
-        self.long = Some(l.trim_start_matches(|c| c == '-'));
+    pub fn long<T: Into<Cow<'static, str>>>(mut self, l: T) -> Self {
+        // NEED_REVIEW
+        self.long = Some(match l.into() {
+            Cow::Borrowed(s) => Cow::Borrowed(s.trim_start_matches(|c| c == '-')),
```

Comment:
Why not just directly trim `"--"` from the start since long must be `--` IIRC?

Reply:
Here is the deal: you give me the working code that does exactly what you said, and I put your credentials to the codebase, accompanied by praises and immodest compliments.

Reply:
I think @pickfire means `s.trim_start_matches("--")`

Reply:
@CreepySkeleton credentials? Wow! Looks interesting but I don't get what you meant.

@pksunkara Correct, `s.trim_start_matches("--")`, just wondering why not this one?

Reply:
>  just wondering why not this one?

Because we would like to strip all the dashes. Your variant will leave one dash in case there's  odd number of them.

> credentials? Wow! Looks interesting but I don't get what you meant.

Your nickname, email, name, whatever you'd prefer to be adressed with; I recommend something in the spirit of "The Emperor of Programming and Button-pressing in general".